_posts/2023-12-12-november-rubygems-updates.md

```diff
@@ -0,0 +1,96 @@
+- added missing `--prefer-local` to Synopsis in bundle-install.1.ronn - [#7194](https://github.com/rubygems/rubygems/pull/7194).
+- allow auto-install to install missing git gems - [#7197](https://github.com/rubygems/rubygems/pull/7197).
+- ensure to require `rubygems` explicitly -  [#7139](https://github.com/rubygems/rubygems/pull/7139).
+
+
+In November, RubyGems gained [110 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-11-01%7D...master@%7B2023-11-31%7D) contributed by 22 authors. There were 1,868 additions and 56,823 deletions across 1,485 files.
+
+## RubyGems.org News
+
+The updates to RubyGems.org this month reflect a strong commitment to improving user experience, enhancing security, and modernizing the platform. Here’s a brief overview of the key improvements in the release:
+
+- fixed deletion `version_id` backfill when a user is deleted - [#4259](https://github.com/rubygems/rubygems.org/pull/4259).
+- upgrade to Postgresql 12 and updated production and staging `DATABASE_URL` to PostgreSQL 12 instance - [#4245](https://github.com/rubygems/rubygems.org/pull/4257), [#4256](https://github.com/rubygems/rubygems.org/pull/4256).
+- added `version_id` column to Deletions - [#4254](https://github.com/rubygems/rubygems.org/pull/4257).
+- added trusted publishers - [#4239](https://github.com/rubygems/rubygems.org/pull/4239).
+- fixed `api_key_created email` when API key belongs to an OIDC id token - [#4233](https://github.com/rubygems/rubygems.org/pull/4233).
+- added `maintenance_task` to backfill info files into S3 - [#4232](https://github.com/rubygems/rubygems.org/pull/4232).
+- used an uncached query to compute compact index info in jobs - [#4231](https://github.com/rubygems/rubygems.org/pull/4231)
+- added job to refresh all OIDC provider configs every 30m #4211
+- extracted verified session logic into a concern - [#4210](https://github.com/rubygems/rubygems.org/pull/4210).
+- updated installation instructions OS X - [#4203](https://github.com/rubygems/rubygems.org/pull/4257).
+- upgraded shoryuken to version 5.x. - [#4166](https://github.com/rubygems/rubygems.org/pull/4166).
+
+
+![Trusted Publishers Demo](https://private-user-images.githubusercontent.com/1946610/289184337-b025c966-f4c2-45df-926a-1f72048e8fc3.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0NTU2NjEsIm5iZiI6MTcwMjQ1NTM2MSwicGF0aCI6Ii8xOTQ2NjEwLzI4OTE4NDMzNy1iMDI1Yzk2Ni1mNGMyLTQ1ZGYtOTI2YS0xZjcyMDQ4ZThmYzMuZ2lmP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTNUMDgxNjAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2MxNDQ0ZWI2NzUzMWFhYTM1YmRlOTlhZDNmYWM3MDEyMGRhM2I4NTZkYjdhZjMxZDIyMzI5ODIyZDc0ZjRiYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.mW0_pyeU3njaWbcWNFQAnyFim5nlZirsJT9MqerSUKk)
```

Comment:
instead let's link to the blog post? https://github.com/rubygems/rubygems.github.io/pull/173

Reply:
https://blog.rubygems.org/2023/12/14/trusted-publishing.html

Reply:
Thanks @segiddins! I will link to the blog post.